node/test/taskissuecommandtests.ts

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import assert = require('assert');
+import * as testutil from './testutil';
+import * as tl from '../_build/task';
+import { IssueSource, _loadData } from '../_build/internal';
+
+
+describe('Task Issue command test', function () {
+
+    before(function (done) {
+        try {
+            testutil.initialize();
+        } catch (err) {
+            assert.fail('Failed to load task lib: ' + err.message);
+        }
+
+        process.env['TASK_SDK_COMMAND_TOKEN'] = 'test_token123';
+        _loadData();
+        done();
+    });
+
+    after(function (done) {
+        delete process.env['TASK_SDK_COMMAND_TOKEN'];
+        _loadData();
+        done();
+    });
+
+    it('removes the token from env var', function (done) {
+        this.timeout(1000);
+
+        assert.equal(process.env['TASK_SDK_COMMAND_TOKEN'], undefined);
+
+        done();
+    })
+
+    it('doesn\'t provide the token using task variables', function (done) {
+        this.timeout(1000);
+
+        process.env['AGENT_VERSION'] = '2.115.0'
+        let variable = tl.getVariable('TASK_SDK_COMMAND_TOKEN');
+        let taskVariable = tl.getTaskVariable('TASK_SDK_COMMAND_TOKEN');
+        assert.equal(variable, undefined);
+        assert.equal(taskVariable, undefined);
+        
+        done();
+    })
+
+    it('adds the token for task.issue messages', function (done) {
```

Comment:
what if token wasn't found in env?

Reply:
If the token wasn't found, it would be absent in the command property list.

Just like this:
https://github.com/microsoft/azure-pipelines-task-lib/blob/f1481180c4673ed739197a571ce9544bd3cc8d47/node/test/resulttests.ts#L50

Reply:
could you please cover this case by test as well (and for powershell)

Reply:
Okay.

That's already implemented for PowerShell with and without a token using
```
powershell/Tests/L0/Write-LoggingCommand.WritesTaskIssue.ps1
powershell/Tests/L0/Write-LoggingCommand.WritesTaskIssueWithToken.ps1
```

Reply:
Done